e2e-tests/functions

```diff
@@ -1258,6 +1258,7 @@ function check_pxc_liveness() {
 	wait_cluster_consistency "${cluster}" "${cluster_size}"
 
 	wait_for_running "${cluster}-pxc" "${cluster_size}"
+	sleep 120
```

Comment:
Do we need `sleep 120` in a function which checks liveness? - either it is alive or not. Maybe we need to add another wait_for_running or something like that.